book/from_flat_php_to_symfony2.rst

```diff
@@ -357,7 +357,7 @@ on the requested URI::
     } elseif ('/index.php/show' === $uri && isset($_GET['id'])) {
         show_action($_GET['id']);
     } else {
-        header('Status: 404 Not Found');
+        header('HTTP/1.0 404 Not Found');
```

Comment:
Yes `HTTP/1.1` is perfectly working and is better than `HTTP/1.0` I will propose it, thanks.
You can follow [this](http://stackoverflow.com/q/246859/4134066) link for more details to know the differences between `HTTP/1.1` and `HTTP/1.0`.